webextensions/README.md

```diff
@@ -0,0 +1,36 @@
+# browser-compat-data
+
+This folder contains compatibility data for the [WebExtensions](https://developer.mozilla.org/Add-ons/WebExtensions) APIs.
+
+It's a JSON file, structured as follows:
+
+* one top level JSON object, which has one property for each WebExtensions API. The name of the property is the name of the module (for example, `"i18n"`, or `"downloads"`). The value is an `api` object.
+
+* an `api` object has one property for each API element (each method, property, type, and event). The name of the property is the name of the element. For example, `"getMessage"` or `"create"`. The value is an `api-element` object.
+
+* an `api-element` is an object with two properties, named: `"desktop_browsers"` and `"mobile_browsers"`. Each of these is a `browser-support` object.
```

Comment:
What if we want more than two levels? E.g., `windows.create.state`, or `storage.managed.getBytesInUse`?